appserver/admingui/pom.xml

```diff
@@ -145,7 +145,8 @@
         <module>hipchat-notifier-console-plugin</module>
         <module>eventbus-notifier-console-plugin</module>
     	<module>email-notifier-console-plugin</module>
-  </modules>
+        <module>SNMP-notifier-console-plugin</module>
```

Comment:
Since you'll be changing the folder name, you'll need to change this to match the new folder name (should be snmp-notifier-console-plugin)